course/migrations/0021_userquestionjunction_is_favourite.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.14 on 2021-10-08 03:05
+
+from django.db import migrations, models
```

Comment:
roll-back the changes from other PRs.

Reply:
So you want me to remove this line?

Reply:
Never mind I read the next comment and I understand what rollback is haha :p

Reply:
I rolled back.